pkg/cli/bootstrap.go

```diff
@@ -72,6 +72,15 @@ func overrideConfigLoaders(honeycombAPIKey, dataset string, tracingDebug bool) {
 					SamplePercent: 100,
 				},
 			}
+			traceConfig.Otel = trace.Otel{
+				Enabled:       traceConfig.Honeycomb.Enabled,
+				Endpoint:      traceConfig.Honeycomb.APIHost,
```

Comment:
You will need to set this to `api.honeycomb.io` manually. Otel automatically provides the scheme and will fail if it's also provided as part of the endpoint config.

Reply:
Thanks @artemchernyak-outreach .  Do you mean just like this? 
```suggestion
				Endpoint:      "api.honeycomb.io",
```

Reply:
Yup, exactly like that. I probably should have have just done a change suggestion on that one.